packages/js/product-editor/src/components/iframe-editor/header-toolbar/more-menu/more-menu.tsx

```diff
@@ -11,12 +12,17 @@ import { MoreMenuDropdown } from '@wordpress/interface';
  * Internal dependencies
  */
 import { ToolsMenuGroup } from './tools-menu-group';
+import { WritingMenu } from '../writing-menu';
+import { getGutenbergVersion } from '../../../../utils/get-gutenberg-version';
 
 export const MoreMenu = () => {
+	const renderBlockToolbar =
+		isWpVersion( '6.5', '>=' ) || getGutenbergVersion() > 17.3;
 	return (
 		<MoreMenuDropdown>
 			{ () => (
 				<>
+					{ renderBlockToolbar && <WritingMenu /> }
```

Comment:
Added the writing menu, just the **Top toolbar** option, as with some of the latest GB changes this setting is reflected across all the editors, so decided to do this here as well instead of always keeping it fixed.